ta/pkcs11/src/pkcs11_token.c

```diff
@@ -14,6 +14,7 @@
 
 #include "pkcs11_token.h"
 #include "pkcs11_helpers.h"
+#include "serializer.h"
```

Comment:
This header doesn't seem to be needed yet.

Reply:
right, this belongs to a former change. will remove.

Reply:
Actually I was wrong. `serialzer.h` is needed by `entry_ck_slot_info()` below.